easy_my_coop_es/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': "Easy My Coop Spain",
+    'version': '12.0.0.0.1',
+    'depends': ['easy_my_coop', 'easy_my_coop_website'],
+    'author': "Coopdevs Treball SCCL",
+    'category': "Cooperative management",
+    'description': """
+    Easy My Coop localization for Spain
+    """,
+    "license": "AGPL-3",
+    'data': [
+        "views/become_cooperator_view.xml",
+        "views/subscription_request_view.xml",
+    ],
+}
```

Comment:
could you add installable = True ?

Reply:
In Odoo 12 the default value of `installable` manifest key is `True`:

https://www.odoo.com/documentation/12.0/reference/module.html

This was changed with Odoo 12 because in Odoo 11 the default value was `False`.

Do you think that it's also needed?

I make a commit with the change, but if it's not necessary we can remove it and merge a cleaner manifest.

Reply:
In general explicit is better than implicit. This way we can see with a quick look if a module is installable.

Thank you Dany

Reply:
> In general explicit is better than implicit. This way we can see with a quick look if a module is installable.
> 
> Thank you Dany

Totally agree!